wgsl/index.bs

```diff
@@ -331,7 +331,7 @@ TODO: What indicates the end of a line?  (E.g. A line ends at the next linefeed
     <tr><th>Token<th>Definition
   </thead>
   <tr><td>`DECIMAL_FLOAT_LITERAL`<td>`(-?[0-9]*.[0-9]+ | -?[0-9]+.[0-9]*)((e|E)(+|-)?[0-9]+)?`
-  <tr><td>`HEX_FLOAT_LITERAL`<td>`-?0x([0-9a-fA-F]*.?[0-9a-fA-F]+ | [0-9a-fA-F]+.[0-9a-fA-F]*)(p|P)(+|-)?[0-9]+`
```

Comment:
A reduced grammar is:

>-?0x *parentheses_dealing_with_dot* (p|P)(+|-)?[0-9]+

So a hex float literal always requires specification of the power. So `0x1` is an int literal.